tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -424,4 +425,34 @@ public function userRemovesSharePermissionOfAResourceInLinkShareUsingGraphAPI(
 			$this->removeSharePermission($sharer, 'link', $resourceType, $resource, $space)
 		);
 	}
+
+	/**
+	 * @Then /^for user "([^"]*)" the space Shares should (not|)\s?contain these (files|entries):$/
+	 *
+	 * @param string $user
+	 * @param string $shouldOrNot
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function forUserTheSpaceSharesShouldContainTheseEntries(string $user, string $shouldOrNot, TableNode $table): void {
+		$should = $shouldOrNot !== 'not';
+		$rows = $table->getRows();
+		$response = GraphHelper::getSharesSharedWithMe(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$user,
+			$this->featureContext->getPasswordForUser($user)
+		);
+		$contents = \json_decode($response->getBody()->getContents(), true);
+
+		$fileFound = !empty(array_intersect(array_map(fn ($row) => trim($row[0], '/'), $rows), array_column($contents['value'], 'name')));
```

Comment:
if we have multiple files to check, this should help
```suggestion
		$fileFound = empty(array_diff(array_map(fn ($row) => trim($row[0], '/'), $rows), array_column($contents['value'], 'name')));
```